src/sql/_testing.py

```
@@ -198,6 +198,8 @@ def database_ready(
             eng.close()
             print(f"{database} is initialized successfully")
             return True
+        except ModuleNotFoundError:
+            raise
```

Do we want to perform any additional operations here?

---

no, we just need to raise the original exception. this allows the tests to fail quickly if they have missing dependencies

---

got it!